### PR TITLE
Fixing Project Warnings

### LINKED ISSUE #49: Fix iOS 9 Warnings
Since we're bumping the deployment target, new warnings have shown up.

Let's get them gone!

Opened by @jleandroperez

### PR DESCRIPTION
**Pull request by jleandroperez**
### Details:
In this PR we're (finally) fixing Xcode's iOS 9 warnings. We're:

- Replacing `SPAddressBookManager` with `SPContactsManager`, based on the new (and shiny) CNContacts Framework
- Updated the SPAddCollaboratorsViewController, to use the new helper
- Removed +20 warnings in one shot (yay!)

### Testing:
1. Launch Simplenote
2. Open any of your favorite notes
3. Tap the top (i) button
4. Press **Collaborate**

Please, test **everything** in there. The ViewController code has been simplified to deal with the new API. Usage is quite simple: just try adding / removing collaborators, via autocomplete, and via the (+) button.

Needs Review: @roundhill / @astralbodies 
Thanks in advance!

Closes #49

### CODE REVIEW

**Comment by jleandroperez** [comment]
@astralbodies already fixed those (or... that's what i suppose!). Please, hit `pod install`, i've pushed a new Simperium release (couple weeks ago) to deal with that.

Thanks!

**Comment by jleandroperez** [comment]
P.s.: The `deleted` warning, yes, i'm afraid that one is (still) expected.

File: Simplenote/Classes/SPContactsManager.swift [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    }
+
+
+    /// Deinitializer
+    ///
+    deinit {
+        NotificationCenter.default.removeObserver(self)
+    }
+
+
```
Comment (roundhill) [inline Simplenote/Classes/SPContactsManager.swift]: There's an extra line here and a bit above ^

Comment (jleandroperez) [inline Simplenote/Classes/SPContactsManager.swift]: Thanks for catching that!

**Comment by jleandroperez** [comment]
@roundhill autocapitalization disabled, mind taking a quick peek?

Thanks Dan!!

**Comment by roundhill** [comment]
:shipit:

**Pull request merged**